src/mscorlib/shared/README.md

```diff
@@ -0,0 +1,15 @@
+# .NET Core Runtime Shared Directory
```

Comment:
`.NET Core Runtime Shared Directory` sounds too general. Call it `System.Private.CoreLib Shared Sources` ?